src/app/drive/services/download.service/downloadFileAsBlob.ts

```diff
@@ -0,0 +1,77 @@
+import downloadFileFromBlob from './downloadFileFromBlob';
+
+interface BlobWritable {
+  getWriter: () => {
+    abort: () => Promise<void>;
+    close: () => Promise<void>;
+    closed: Promise<undefined>;
+    desiredSize: number | null;
+    ready: Promise<undefined>;
+    releaseLock: () => void;
+    write: (chunk: BlobPart) => Promise<void>;
+  };
+  locked: boolean;
+  abort: () => Promise<void>;
+  close: () => Promise<void>;
+}
+
+async function getBlobWritable(filename: string, onClose: (result: Blob) => void): Promise<BlobWritable> {
```

Comment:
the async is really necessary? If I'm not mistaken, is not needed in this function